src/cmd/manage_nsfs.js

```diff
@@ -473,7 +474,10 @@ async function update_account(data, is_flag_iam_operate_on_root_account) {
     const data_name = new_name || cur_name;
     data.name = data_name;
     data.email = data_name; // saved internally
-    data.access_keys[0].access_key = data.new_access_key || cur_access_key;
+    data.access_keys[0] = {
+        access_key: data.new_access_key || new SensitiveString(cur_access_key),
```

Comment:
print and check if the sensitive string here is redundant

Reply:
It was redundant and therefore I removed it, thanks.